invenio_app_ils/circulation/views.py

```diff
@@ -144,6 +173,24 @@ def post(self, **kwargs):
         )
 
 
+class BulkLoanExtensionResource(IlsCirculationResource):
+    """Bulk loan extension resource."""
+
+    view_name = "bulk_loan_extension"
+
+    @need_permissions("bulk-loan-extension")
+    def post(self, **kwargs):
+        """Loan checkout post method."""
+        data = self.loader()
+        if "patron_pid" not in data:
```

Comment:
It might that the `patron_pid` sent is a wrong one. Maybe safer to set it to the current logged in user or failed if different?

Reply:
I added the check below now